plugins/modules/one_vm.py

```diff
@@ -1628,11 +1628,11 @@ def get_connection_info(module):
             if authfile is None:
                 authfile = os.path.join(os.environ.get("HOME"), ".one", "one_auth")
             try:
-                with open(authfile, "r") as fp:
+                with open(authfile) as fp:
```

Comment:
As I mentioned before I don't think `open(filename)` is great since it's not clear whether it opens the file in binary or text mode, but removing the `"r"` here doesn't change that ambiguity, so this change is fine for me.

(And yes, this is mainly a problem for folks like me who handle too many different programming languages and can't keep *every* little detail in their head ;-) )